khal/utils.py

```diff
@@ -37,12 +38,14 @@
 
 
 def timefstr(dtime_list, timeformat):
-    """converts a time (as a string) to a datetimeobject
```

Comment:
Unrelated change, also why is that API designed like that? Why not take just a time as a string? Why does it delete the item from the list?